src/main/res/values-fr/strings.xml

```diff
@@ -410,31 +390,6 @@
     <string name="calypso_manufacture_country">Pays de fabrication</string>
     <string name="manufacture_date">Date de fabrication</string>
     <string name="manufacturer_name">Fabricant</string>
-    <string name="calypso_manufacturer_ask">ASK</string>
```

Comment:
For `calypso_manufacturer_`, should this be marked as "not translatable"?  Some the translations have partially translated company names, but I'm not familiar enough with the entities involved to know whether they change their names in different marketing materials.

I noticed `calypso_manufacturer_acg` is translated in some places, but I can't see evidence that they change their name in any of their marketing materials.

It's also probably not reasonable to expect these to be researched by every translator :)

Reply:
I've now fixed that one, they are marked as "not translatable" and the extra translations are now deleted.

Reply:
I figured out what happened here. 5d0aa63defee74ddcf3acdebe6bcc691754b330c converted it from StringResource to `string` again...

Reply:
And now I fixed that in 63a88c02.